### PR TITLE
[15.0] [MIG] mis_builder_operating_unit

### PR DESCRIPTION
**Pull request by Rajan-SCS**

### CODE REVIEW

**Review by AaronHForgeFlow — COMMENTED** [review]
Can you please re-license to AGPL? The base module mis_builder is AGPL

**Comment by Rajan-SCS** [comment]
@AaronHForgeFlow license updated so, please check.

File: mis_builder_operating_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# © 2015-19 ForgeFlow S.L. -
```
Comment (AaronHForgeFlow) [inline mis_builder_operating_unit/__manifest__.py:1]:
```suggestion
# Copyright 2015-19 ForgeFlow S.L. -
```

**Comment by AaronHForgeFlow** [comment]
License is fine now, thanks, can you check CI failing checks?

**Comment by AaronHForgeFlow** [comment]
Superseded in here:
https://github.com/OCA/operating-unit/pull/587

**Pull request closed without merging**